actions/supr.py

```diff
@@ -50,7 +50,8 @@ def search_for_pis(project_to_email_dict, supr_base_api_url, api_user, api_key):
     def create_delivery_project(base_url, project_names_and_ids, staging_info, project_info, user, key):
 
         result = {}
-        for ngi_project_name, pi_id in project_names_and_ids.iteritems():
```

Comment:
Any particular reason for this change? I think that it should be equivalent to use ´iteritems´ to using ´keys´ and then getting the value?